src/main/webapp/app/entities/static-code-analysis-issue.model.ts

```diff
@@ -0,0 +1,11 @@
+export class StaticCodeAnalysisIssue {
+    public filePath: string;
+    public startLine: number;
+    public endLine: number;
+    public startColumn: number | null;
```

Comment:
We would like to avoid null values in the future and use the following instead.
I am currently preparing this in another branch, would be nice if you can already adopt it.

Reply:
```suggestion
    public startColumn?: number;
```

Reply:
Basically all elements coming from the server should use the `?` to emphasize that they could be undefined, in case the json payload does not include them

Reply:
Adopted the requested change